cns/grpc/server.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	pb "grpc/protogen"
+	"go.uber.org/zap"
+)
+
+// Server struct to hold the gRPC server settings and the CNS service.
+type Server struct {
+	Settings   GrpcServerSettings
+	CnsService pb.CNSServer
+	Logger     *zap.Logger
+}
+
+// GrpcServerSettings holds the gRPC server settings.
+type GrpcServerSettings struct {
+	IPAddress string
+	Port      uint
```

Comment:
Nit: ports are defined as uint16s

Reply:
Hey Tim, just implemented this change thank you!